src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -222,6 +222,7 @@ private void HandleMessage(Message message, IRequest request, IIssueConsumer con
                 CmdLineArgs = new[] { communicateViaStreaming },
                 CancellationToken = cancellationToken,
                 WorkingDirectory = SubProcessFilePaths.WorkingDirectory,
+                EnvironmentVariables = request.EnvironmentVariables,
```

Comment:
Fortunately, the `ProcessRunnerArguments` already supported environment variables, even though we weren't using them (we copied the runner and arguments from the Scanner for MSBuild, which does pass environment variables, and fortunately we didn't remove the property).